.github/workflows/ci.yml

```diff
@@ -30,10 +30,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        runs-on: [ubuntu-22.04, windows-2022, macos-13]
+        runs-on: [ubuntu-24.04, windows-2022, macos-14]
```

Comment:
Wow, you're bold! (doing this in the same PR)

🍀

Reply:
I'll probably pull out the changes that work (22->24) if this gets stuck. Looks like 13->14 might cause updates to the pins, but that's likely to be needed for 3.14 too. But it looks like there's at least a little trouble.